src/pages/data-feeds/price-feeds/addresses.mdx

```diff
@@ -6,8 +6,8 @@ metadata:
   title: "Price Feed Contract Addresses"
   description: "A list of Price Feed addresses on supported networks."
 date: Last Modified
-setup: |
-  import FeedPage from "@features/feeds/components/FeedPage.astro"
 ---
 
-<FeedPage  />
+import FeedPage from "@features/feeds/components/FeedPage.astro"
```

Comment:
Somehow a slightly different CSS is getting applied in the MDX version. The network selectors are bunched close together and need more padding.

Preview: https://documentation-git-fork-thedriftofwords-mdx-a2471c-chainlinklabs.vercel.app/data-feeds/price-feeds/addresses

Current page: https://docs.chain.link/data-feeds/price-feeds/addresses

![image](https://user-images.githubusercontent.com/8083094/226808299-5da01c19-4ab5-4b42-b65a-35b31ee1fa3d.png)

Reply:
It appears normal in my local environment, where I still have Astro 2.0.2 installed. So I expect this problem to go away when we update the package.json and configs. I can revert this specific file for now and then add it to PR #1189.

Reply:
Reminder: 

- The same issue also affects the Proof of Reserve addresses page (https://docs.chain.link/data-feeds/proof-of-reserve/addresses). 
- The NFT Floor Price one (https://docs.chain.link/data-feeds/nft-floor-price/addresses) only has one network selector right now.

Reply:
I've handled this by reverting the addresses files to .md. We can change them back to .mdx as part of #1189